classes/Hook/HookActionValidateOrder.php

```diff
@@ -53,8 +53,9 @@ public function __construct(Ps_Googleanalytics $module, Context $context)
      */
     public function run()
     {
-        // Check if we are creating backoffice order
-        if ($this->context->controller->controller_name != 'AdminOrders' && $this->context->controller->controller_name != 'Admin') {
+        // Check if we are creating backoffice order, we are only launching this hook when creating backoffice order
+        // For FO purposes, we use displayOrderConfirmation.
+        if (empty($this->context->controller->controller_name) || ($this->context->controller->controller_name != 'AdminOrders' && $this->context->controller->controller_name != 'Admin')) {
```

Comment:
maybe this is clearer ?

```suggestion
        if (isset($this->context->controller->controller_name) &&  !in_array($this->context->controller->controller_name, ['Admin', 'AdminOrders']) {
```